packages/ui-components/src/globals/globals.ts

```diff
@@ -0,0 +1,33 @@
+import { setMode } from '@stencil/core';
+import { get } from 'lodash-es';
+import { StyleMode, UIComponentsConfig } from '../types';
+
+let defaultStyleMode: StyleMode = StyleMode.Night;
+
+export const initialize = (userConfig: UIComponentsConfig = {}) => {
+	defaultStyleMode = get(userConfig, 'styleMode', defaultStyleMode);
```

Comment:
instead of creating a global variable why we don't put the `StyleMode.Night` at the end?